### PR TITLE
include Matomo configuration

### PR DESCRIPTION
**Pull request by Beck-berry**
### The snippet
Use the snippet given by Matomo Tag Manager in the getAnalyticsInitCode function. It has an URL inside which should be configurable.

### The URL
Each config (which would like to use MTM) should include the MATOMO_URL environment variable.

### CODE REVIEW

File: app/configurations/config.hb.js [hunk 1]
```diff
@@ -38,6 +38,9 @@ export default configMerger(walttiConfig, {
   availableLanguages: ['de', 'en'],
   defaultLanguage: 'de',
 
+  // Matomo Tag Manager URL
+  MTM_URL: process.env.MTM_URL || 'https://cdn.matomo.cloud/mobilinherrenberg.matomo.cloud/container_jUK6B6kx.js',
```
Comment (hbruch) [inline app/configurations/config.hb.js]: Default should be empty: If nothing is defined, we should not track

File: app/util/analyticsUtils.js [hunk 2]
```diff
@@ -25,9 +25,20 @@ export function addAnalyticsEvent(event) {
...
+  if (MTM_URL) {
+    return `<!-- Matomo Tag Manager -->
+      <script type="text/javascript">
+      var _mtm = _mtm || [];
+      _mtm.push({'mtm.startTime': (new Date().getTime()), 'event': 'mtm.Start'});
+      var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+      g.type='text/javascript'; g.async=true; g.defer=true; g.src='${MTM_URL}'; s.parentNode.insertBefore(g,s);
+      </script>
+      <!-- End Matomo Tag Manager -->`;
+  }
```
Comment (hbruch) [inline app/util/analyticsUtils.js]: Should we really support both at the same time? I'd suggest to use else if instead of if

Comment (Beck-berry) [reply app/util/analyticsUtils.js]: I believed if there is a return in the 'if' section, the other 'ifs' do not have a chance. (ESLint also gives an error with this modification: "unnecessary else after return".) So only the first true statement will have an effect.
Would you like me to do it anyway?

File: app/configurations/config.hb.js [hunk 3]
```diff
@@ -20,6 +20,12 @@ const maxLat = 48.64040;
 const minLon = 8.78597;
 const maxLon = 8.98613;
 
+// Matomo Tag Manager
```
Comment (hbruch) [inline app/configurations/config.hb.js]: What is the advantage of having the matomo url split in it's constituents instead of just MATOMO_URL?

Comment (frankgerhardt) [reply app/configurations/config.hb.js]: if you deploy to staging, the hostname will be different.
in matomo you would track the staging server seperately, in e.g. configured in Matomo as "mib-staging". Then the path in the URL would change. The container_id would also be different.

**Pull request merged**